torchtitan/config/job_config.py

```diff
@@ -408,6 +408,13 @@ class Checkpoint:
     enable: bool = False
     """Whether to enable checkpoint"""
 
+    enable_ft: bool = True
```

Comment:
`enable_ft` is too vague. I got confused that this is a similar thing as FT.enabled. Can we have a longer and more meaningful naming?